### PR TITLE
Added story to forest

### PR DESCRIPTION
**Pull request by keldavis**
Please review this, @casey-collab

### CODE REVIEW

File: english/forest/drag/snake/slither/safety.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+You slither down a nice, warm, tight, wet hole. You feel very safe here.
```
Comment (casey-collab) [inline english/forest/drag/snake/slither/safety.md]: This file has also been added to a directory that already contained a page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by keldavis** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :thumbsup:

**Pull request merged**